tests/i18n/tests.py

```diff
@@ -704,6 +704,7 @@ def test_l10n_disabled(self):
                 self.assertEqual('66666.67', Template('{{ n|floatformat:2 }}').render(self.ctxt))
                 self.assertEqual('100000.0', Template('{{ f|floatformat }}').render(self.ctxt))
 
+    @override_settings(USE_THOUSAND_SEPARATOR=True, THOUSAND_SEPARATOR='!', FIRST_DAY_OF_WEEK=1)
```

Comment:
Had to put this here since the call to `patch_formats` actually modifies the _format_cache, but the new localize_settings_changed signal handler clears out this cache.